cmd/kubectl-gadget/main.go

```diff
@@ -162,7 +162,7 @@ func main() {
 	rootCmd.AddCommand(advise.NewAdviseCmd(gadgetNamespace))
 	rootCmd.AddCommand(NewTraceloopCmd(gadgetNamespace))
 	rootCmd.AddCommand(common.NewSyncCommand(grpcRuntime))
-	rootCmd.AddCommand(common.NewRunCommand(rootCmd, grpcRuntime, hiddenColumnTags))
+	rootCmd.AddCommand(common.NewRunCommand(rootCmd, grpcRuntime, hiddenColumnTags, common.CommandModeRun))
```

Comment:
WDYM? Attaching from `kubectl-gadget` is not handled by this PR, but it will work in the same way this does (although it'll use another store implementation.)

Reply:
It just needs another store implementation (based on Kubernetes resources, so ConfigMap or CR) - that will be done in a follow-up PR.